langmeta-sqlite/src/main/resources/semanticdb.ddl

```diff
@@ -0,0 +1,49 @@
+create table document(
```

Comment:
I think document needs a reference to a new table or a few new columns specifying the conditions under which it was created.
Compiler flags
Dependencies
Scala version

Basically all of the inputs to zinc.  Without this the results may be incorrect.  For example types inferred may change based on dependencies, or messages output may change based on -xlint flag.

This does not need to hold up this pr though.